src/handlers/note.rs

```diff
@@ -0,0 +1,107 @@
+//! Allow users to add summary comments in Issues & Pull Requests.
+//!
+//! Users can make a new summary entry by commenting the following:
+//!
+//! ```md
+//! @rustbot note summary-title
+//!
+//! ...details details details...
+//! ```
+//!
+//! If this is the first summary entry, rustbot will amend the original post (the top-level comment) to add a "Notes" section:
+//!
+//! ```md
+//! <!-- TRIAGEBOT_SUMMARY_START -->
+//!
+//! ### Summary Notes
+//!
+//! - ["summary-title" by @username](link-to-comment)
```

Comment:
Can we add a line below these that says something like `Generated by triagebot, see [help](wiki-page-link) for how to add more`. (And we'll want to create that wiki page https://github.com/rust-lang/triagebot/wiki -- let me know if you need help with that, not sure what permissions are like).